ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -356,24 +412,19 @@ static int
 gcp_object_list_single(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *directory,
```

Comment:
add function comment here and other function implementation where it's missing

Reply:
updated d11e042e49654d91f7ee8bd8450515ee77e1766e